### DISCUSSION TITLE
Can @Retryable work without explicit @EnableRetry?

### DISCUSSION THREAD

**Discussion by lukaszungier** [spring-projects/spring-retry#398] [Q&A]
Hey,
I've encountered a case when `@Retryable` works in of my projects without explicit declaration of `@EnableRetry`, but in some others - it doesn't work.

Is it possible? Could this be enabled in an implicit way? Or maybe it could depend on version of spring-retry or Spring Boot?

Thank you in advance.

**Comment by artembilan** [comment]
The `@Retryable` cannot work by itself: the "hard" infrastructure logic to parse that annotation is indeed covered by the `RetryConfiguration` behind `@EnableRetry`.
There is no auto-configuration for that in Spring Boot (), so you need to dig your project for a cause.
